ra/runtime/src/main/java/io/quarkus/artemis/jms/ra/ArtemisResourceAdapterFactory.java

```diff
@@ -43,7 +43,7 @@ public String getProductVersion() {
     @Override
     public ActiveMQResourceAdapter createResourceAdapter(String id, Map<String, String> config) {
         ActiveMQResourceAdapter adapter = new ActiveMQResourceAdapter();
-        adapter.setConnectorClassName(NettyConnectorFactory.class.getName());
+        adapter.setConnectorClassName(config.getOrDefault("connector-class-name", NettyConnectorFactory.class.getName()));
```

Comment:
@gastaldi @vsevel IIUC, there could be multi ConnectorClassName if more than two hosts are configured. Then I wonder the default valude of `connector-class-name` might be `NettyConnectorFactory` * the number of hosts?

Reply:
I considered that too while creating the PR but I was hoping that you could use a different connector class here. OTOH I wasn't sure if this attribute should be configurable, so I took the easiest path 😀

Reply:
additionally, one idea would be to complete the connector classname string to match the size of the connection parameters.
in other words if I set a connection parameters to set 2 hosts, by I include only one connector class name, then I would automatically add an additional connector class name, to match the number of hosts.
having to match both size is really error prone.

Reply:
or log a warning, or throw an error.

Reply:
Actually I kinda like @zhfeng 's suggestion. I think introducing a new parameter will require proper validation and I don't see a use case requiring it now BTW

Reply:
I went ahead and implemented @zhfeng's suggestion in [`62e01a9` (#572)](https://github.com/quarkiverse/quarkus-artemis/pull/572/commits/62e01a9201112665d8ba9bace7c8ca77cdfae306)